### PR TITLE
DBR-310 - Semantic Release

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

This is to generate a version number and release notes based on our commit messages, as [we are already doing in Form Generator](https://github.com/MakingSense/doppler-forms-generator/releases).

The idea is having better information about what version are we using in the WebApp in each moment.

Then, I will apply the same approach to WebApp and integrate it with docker images generation.

⚠ **After now, all we will need to take care of the commit messages.**

Could you review?

### Example of the first version

```
# 1.0.0 (2019-05-20)

    * Merge pull request #93 from GusBaamonde/paid_account_with_no_trial_activated (538931b (https://github.com/FromDoppler/Doppler-UI-System/commit/538931b)), closes #93 (https://github.com/FromDoppler/Doppler-UI-System/issues/93)

### Bug Fixes

    * add a way to close user menu when clicking outside it (88973e9 (https://github.com/FromDoppler/Doppler-UI-System/commit/88973e9))
    * add compiled files exception to gitignore, removed unused dependencies. (f6c6d3e (https://github.com/FromDoppler/Doppler-UI-System/commit/f6c6d3e))
    * add enter line in confirmation page (fdc4c38 (https://github.com/FromDoppler/Doppler-UI-System/commit/fdc4c38))
    * add reset buttons (6fbd933 (https://github.com/FromDoppler/Doppler-UI-System/commit/6fbd933))
    * add variable for fadeIn velocity. (a041ed6 (https://github.com/FromDoppler/Doppler-UI-System/commit/a041ed6))
    * add variables (26ccce2 (https://github.com/FromDoppler/Doppler-UI-System/commit/26ccce2))
    * Add variables (c15eb7a (https://github.com/FromDoppler/Doppler-UI-System/commit/c15eb7a))
    * add variables and delete unnecessary class (2389045 (https://github.com/FromDoppler/Doppler-UI-System/commit/2389045))
    * apply proximanova font in validation errors (2e63711 (https://github.com/FromDoppler/Doppler-UI-System/commit/2e63711))
    * change color in the border input and placeholder (fed1de2 (https://github.com/FromDoppler/Doppler-UI-System/commit/fed1de2))
    * colors variables (bdf2380 (https://github.com/FromDoppler/Doppler-UI-System/commit/bdf2380))
    * colors variables (93f8426 (https://github.com/FromDoppler/Doppler-UI-System/commit/93f8426))
    * colors variables and remove extra spaces in js (cb75119 (https://github.com/FromDoppler/Doppler-UI-System/commit/cb75119))
    * create variable fadeIn (c702e89 (https://github.com/FromDoppler/Doppler-UI-System/commit/c702e89))
    * editor configs conventions (92fd7fb (https://github.com/FromDoppler/Doppler-UI-System/commit/92fd7fb))
    * editorconfig requirements (c4dc551 (https://github.com/FromDoppler/Doppler-UI-System/commit/c4dc551))
    * elements forms (6b83cf0 (https://github.com/FromDoppler/Doppler-UI-System/commit/6b83cf0))
    * fix fonts and icons (e635761 (https://github.com/FromDoppler/Doppler-UI-System/commit/e635761))
    * fix resolution for signup in 768px resolutions (28e7f4e (https://github.com/FromDoppler/Doppler-UI-System/commit/28e7f4e))
    * fix styles in the signup (8de3a09 (https://github.com/FromDoppler/Doppler-UI-System/commit/8de3a09))
    * font size list country (b4a8fee (https://github.com/FromDoppler/Doppler-UI-System/commit/b4a8fee))
    * general adjustments of margins and paddings for 320px (edf5096 (https://github.com/FromDoppler/Doppler-UI-System/commit/edf5096))
    * header language (c4db095 (https://github.com/FromDoppler/Doppler-UI-System/commit/c4db095))
    * header nav right to avoid inherit ul styles (a7d573b (https://github.com/FromDoppler/Doppler-UI-System/commit/a7d573b))
    * I deleted the phrase and put the show password icon (7e71a99 (https://github.com/FromDoppler/Doppler-UI-System/commit/7e71a99))
    * issues with icon positioning (cdf3857 (https://github.com/FromDoppler/Doppler-UI-System/commit/cdf3857))
    * issues with line-height and font declaration specificity (bb0e8ea (https://github.com/FromDoppler/Doppler-UI-System/commit/bb0e8ea))
    * link--title is uppercase (82ca989 (https://github.com/FromDoppler/Doppler-UI-System/commit/82ca989))
    * links should inherit font-size (b156ddb (https://github.com/FromDoppler/Doppler-UI-System/commit/b156ddb))
    * margins and paddings header (9f0160e (https://github.com/FromDoppler/Doppler-UI-System/commit/9f0160e))
    * messages header (6b9a72d (https://github.com/FromDoppler/Doppler-UI-System/commit/6b9a72d))
    * mixin transition name (it was crashing the build) (4128978 (https://github.com/FromDoppler/Doppler-UI-System/commit/4128978))
    * modal interaction (f1604ad (https://github.com/FromDoppler/Doppler-UI-System/commit/f1604ad))
    * modify content in confirmation page (2646c9a (https://github.com/FromDoppler/Doppler-UI-System/commit/2646c9a))
    * modify space entry inputs the two columns and delete // not necessary (6a2328d (https://github.com/FromDoppler/Doppler-UI-System/commit/6a2328d))
    * modify styles, logos, validations styles and add eye to show password (1eb8c36 (https://github.com/FromDoppler/Doppler-UI-System/commit/1eb8c36))
    * normalize spaces in login and signup (21139a2 (https://github.com/FromDoppler/Doppler-UI-System/commit/21139a2))
    * normalize spaces in login and signup (d569201 (https://github.com/FromDoppler/Doppler-UI-System/commit/d569201))
    * remove commented line. (a0dd5fc (https://github.com/FromDoppler/Doppler-UI-System/commit/a0dd5fc))
    * remove commented lines. Gitignore file modified. (126f9b9 (https://github.com/FromDoppler/Doppler-UI-System/commit/126f9b9))
    * remove forced ip location (c058ee3 (https://github.com/FromDoppler/Doppler-UI-System/commit/c058ee3))
    * replace button components by new ones based on reports mockup (96ca7de (https://github.com/FromDoppler/Doppler-UI-System/commit/96ca7de))
    * reset correct styles in signup (8855f0d (https://github.com/FromDoppler/Doppler-UI-System/commit/8855f0d))
    * separate button inside the form (bbddb90 (https://github.com/FromDoppler/Doppler-UI-System/commit/bbddb90))
    * background: correction in background images (b8d72a2 (https://github.com/FromDoppler/Doppler-UI-System/commit/b8d72a2))
    * build: delete functions that no use (0c01b44 (https://github.com/FromDoppler/Doppler-UI-System/commit/0c01b44))
    * build: delete watching change in build-w-docker and change task to run gulpfile correctly (63f5220 (https://github.com/FromDoppler/Doppler-UI-System/commit/63f5220))
    * build: fixed generation the fonts in gulpfile (0dc54e4 (https://github.com/FromDoppler/Doppler-UI-System/commit/0dc54e4))
    * build: in build-w-docker no use -rm dev and gulpfile no use function sass:build because this delete (67a079a (https://github.com/FromDoppler/Doppler-UI-System/commit/67a079a))
    * gulpfile: remove unused code and fix dist task (5cd8576 (https://github.com/FromDoppler/Doppler-UI-System/commit/5cd8576))
    * signup: fix component the signup (eed6e11 (https://github.com/FromDoppler/Doppler-UI-System/commit/eed6e11))
    * signup: modification component destination, modification nomenclature class and add class level space (c53ae81 (https://github.com/FromDoppler/Doppler-UI-System/commit/c53ae81))
    * styles: correction tags and class (58d0684 (https://github.com/FromDoppler/Doppler-UI-System/commit/58d0684))

### Features

    * a mixin was created for the reports boxes (ef626ff (https://github.com/FromDoppler/Doppler-UI-System/commit/ef626ff))
    * add a class for the registered image and change the property so that do not see the text (6ec04ae (https://github.com/FromDoppler/Doppler-UI-System/commit/6ec04ae))
    * add a validation to the password field (cdf1a9d (https://github.com/FromDoppler/Doppler-UI-System/commit/cdf1a9d))
    * add atomic class for text italic and add more level space (7907954 (https://github.com/FromDoppler/Doppler-UI-System/commit/7907954))
    * add atomic class for text italic and add more level space (4e157b1 (https://github.com/FromDoppler/Doppler-UI-System/commit/4e157b1))
    * add button-loading and animation (3357cce (https://github.com/FromDoppler/Doppler-UI-System/commit/3357cce))
    * add class error (fbb4422 (https://github.com/FromDoppler/Doppler-UI-System/commit/fbb4422))
    * add class for modal button layout (6db4e5c (https://github.com/FromDoppler/Doppler-UI-System/commit/6db4e5c))
    * add class for validate form (21805a7 (https://github.com/FromDoppler/Doppler-UI-System/commit/21805a7))
    * add class in html (3d0f27f (https://github.com/FromDoppler/Doppler-UI-System/commit/3d0f27f))
    * add class primary-grey for button (f3c857e (https://github.com/FromDoppler/Doppler-UI-System/commit/f3c857e))
    * add class to make a proof of concept (410c5c3 (https://github.com/FromDoppler/Doppler-UI-System/commit/410c5c3))
    * add classes for backgrounds images (4573582 (https://github.com/FromDoppler/Doppler-UI-System/commit/4573582))
    * add classes to validate form. (ada5bc9 (https://github.com/FromDoppler/Doppler-UI-System/commit/ada5bc9))
    * add confirmation page (5bab59f (https://github.com/FromDoppler/Doppler-UI-System/commit/5bab59f))
    * add elements and class html (1eb0e6f (https://github.com/FromDoppler/Doppler-UI-System/commit/1eb0e6f))
    * add elements for html (9fda765 (https://github.com/FromDoppler/Doppler-UI-System/commit/9fda765))
    * add elements html top ranking (c861c59 (https://github.com/FromDoppler/Doppler-UI-System/commit/c861c59))
    * Add flags image, styles and js files for new intel-input library (89dba5f (https://github.com/FromDoppler/Doppler-UI-System/commit/89dba5f))
    * add footer component (158ea0b (https://github.com/FromDoppler/Doppler-UI-System/commit/158ea0b))
    * add header component and foundation colors (3f35025 (https://github.com/FromDoppler/Doppler-UI-System/commit/3f35025))
    * add html forgot password (7f02905 (https://github.com/FromDoppler/Doppler-UI-System/commit/7f02905))
    * add icon-font (f2ce878 (https://github.com/FromDoppler/Doppler-UI-System/commit/f2ce878))
    * add icons for layout reports (a6f0e8d (https://github.com/FromDoppler/Doppler-UI-System/commit/a6f0e8d))
    * add image the background and icon card the registration (db73e80 (https://github.com/FromDoppler/Doppler-UI-System/commit/db73e80))
    * add input styles (e350b0f (https://github.com/FromDoppler/Doppler-UI-System/commit/e350b0f))
    * add js for phone input (2213f89 (https://github.com/FromDoppler/Doppler-UI-System/commit/2213f89))
    * add link in the logo (ad69db4 (https://github.com/FromDoppler/Doppler-UI-System/commit/ad69db4))
    * add links in pages (c33694b (https://github.com/FromDoppler/Doppler-UI-System/commit/c33694b))
    * Add mixin for preloaders (82e478f (https://github.com/FromDoppler/Doppler-UI-System/commit/82e478f))
    * add navigation for doppler ui (01e1f80 (https://github.com/FromDoppler/Doppler-UI-System/commit/01e1f80))
    * add parameters for max-width and min-height (07998a2 (https://github.com/FromDoppler/Doppler-UI-System/commit/07998a2))
    * add parameters in include (9143a0d (https://github.com/FromDoppler/Doppler-UI-System/commit/9143a0d))
    * add space in end of file (7070885 (https://github.com/FromDoppler/Doppler-UI-System/commit/7070885))
    * add spacing (8bcf548 (https://github.com/FromDoppler/Doppler-UI-System/commit/8bcf548))
    * add style for button disabled (b27a16f (https://github.com/FromDoppler/Doppler-UI-System/commit/b27a16f))
    * add style for hero filters (51966a0 (https://github.com/FromDoppler/Doppler-UI-System/commit/51966a0))
    * add styles for modal and form user-menu (88b414c (https://github.com/FromDoppler/Doppler-UI-System/commit/88b414c))
    * add styles for top ranking (47af299 (https://github.com/FromDoppler/Doppler-UI-System/commit/47af299))
    * add text uppercase (0e4c285 (https://github.com/FromDoppler/Doppler-UI-System/commit/0e4c285))
    * add uppercase atomic class (91cbf0b (https://github.com/FromDoppler/Doppler-UI-System/commit/91cbf0b))
    * add variable (f480076 (https://github.com/FromDoppler/Doppler-UI-System/commit/f480076))
    * add variables (51d39fd (https://github.com/FromDoppler/Doppler-UI-System/commit/51d39fd))
    * add variables (f791a3d (https://github.com/FromDoppler/Doppler-UI-System/commit/f791a3d))
    * add variables (83f9a70 (https://github.com/FromDoppler/Doppler-UI-System/commit/83f9a70))
    * add variables for margin normalization (44add81 (https://github.com/FromDoppler/Doppler-UI-System/commit/44add81))
    * add variables margin generic (3adcaac (https://github.com/FromDoppler/Doppler-UI-System/commit/3adcaac))
    * Added the html elements and login form styles (70a8584 (https://github.com/FromDoppler/Doppler-UI-System/commit/70a8584))
    * Adjust margins to see the create account button on a 650px high screen. (a5fe175 (https://github.com/FromDoppler/Doppler-UI-System/commit/a5fe175))
    * adjustment of margin variables (08e1369 (https://github.com/FromDoppler/Doppler-UI-System/commit/08e1369))
    * Adjustment of width and distribution of boxes (920f7a7 (https://github.com/FromDoppler/Doppler-UI-System/commit/920f7a7))
    * aligning boxes (5e68ece (https://github.com/FromDoppler/Doppler-UI-System/commit/5e68ece))
    * All the variables were put together in a single file for later refactoring (1fa2fef (https://github.com/FromDoppler/Doppler-UI-System/commit/1fa2fef))
    * allow span without p in form-message (46636b3 (https://github.com/FromDoppler/Doppler-UI-System/commit/46636b3))
    * Build the header with the language for the public sections. Sign up, forgot and login. (6e42ca1 (https://github.com/FromDoppler/Doppler-UI-System/commit/6e42ca1))
    * Change for new dynamic mixin for preloads (4a7aaeb (https://github.com/FromDoppler/Doppler-UI-System/commit/4a7aaeb))
    * Change image, signup, login and forgot password. (303cd4e (https://github.com/FromDoppler/Doppler-UI-System/commit/303cd4e))
    * change of images in signup, login and forgot password (b6e08d7 (https://github.com/FromDoppler/Doppler-UI-System/commit/b6e08d7))
    * change of max width (d45551b (https://github.com/FromDoppler/Doppler-UI-System/commit/d45551b))
    * Classes for buttons and classes were added to validate the form in the html. Classes were added to see the modal buttons in the html. (7531e5c (https://github.com/FromDoppler/Doppler-UI-System/commit/7531e5c))
    * classes for kpi were added for report layout (bce9a58 (https://github.com/FromDoppler/Doppler-UI-System/commit/bce9a58))
    * code spacing (fda087a (https://github.com/FromDoppler/Doppler-UI-System/commit/fda087a))
    * comment added in the script that warns only for demo use (cbe9773 (https://github.com/FromDoppler/Doppler-UI-System/commit/cbe9773))
    * correction and alignment of boxes in the report layout (b4d1326 (https://github.com/FromDoppler/Doppler-UI-System/commit/b4d1326))
    * create a container for unexpected messages (7729b6f (https://github.com/FromDoppler/Doppler-UI-System/commit/7729b6f))
    * Create container to display datahub messages (6960856 (https://github.com/FromDoppler/Doppler-UI-System/commit/6960856))
    * create html for preload (bde0444 (https://github.com/FromDoppler/Doppler-UI-System/commit/bde0444))
    * create preload page and animation (cfa9c2d (https://github.com/FromDoppler/Doppler-UI-System/commit/cfa9c2d))
    * create separate templates for each component (01f3fc9 (https://github.com/FromDoppler/Doppler-UI-System/commit/01f3fc9))
    * created preload for layout reports (1cec136 (https://github.com/FromDoppler/Doppler-UI-System/commit/1cec136))
    * creation of a file of atomic and variable classes for its implementation and normalization of spaces. (539fa78 (https://github.com/FromDoppler/Doppler-UI-System/commit/539fa78))
    * creation of a file of atomic classes for its implementation and normalization of spaces (d323cd4 (https://github.com/FromDoppler/Doppler-UI-System/commit/d323cd4))
    * delete backgrounds that not use (734f1c0 (https://github.com/FromDoppler/Doppler-UI-System/commit/734f1c0))
    * delete margin class in HTML and use in the element (1e80968 (https://github.com/FromDoppler/Doppler-UI-System/commit/1e80968))
    * fix padding top ranking (db1d8af (https://github.com/FromDoppler/Doppler-UI-System/commit/db1d8af))
    * homogeneous visual coherence for messages and alignment of footer boxes (4bb5529 (https://github.com/FromDoppler/Doppler-UI-System/commit/4bb5529))
    * I added external scripts and css for a correct visualization. (a9e502b (https://github.com/FromDoppler/Doppler-UI-System/commit/a9e502b))
    * I removed the old font size variables, replacing them with new variables in levels, impacting all the scss files. I've talked to Santiago and he agrees. (eb3fdad (https://github.com/FromDoppler/Doppler-UI-System/commit/eb3fdad))
    * messages site tracking user data (f1f6ed6 (https://github.com/FromDoppler/Doppler-UI-System/commit/f1f6ed6))
    * messages that validate the password (420f42c (https://github.com/FromDoppler/Doppler-UI-System/commit/420f42c))
    * modification of variables for reuse (f13abab (https://github.com/FromDoppler/Doppler-UI-System/commit/f13abab))
    * Modify the position of the password safe icon (4ed1095 (https://github.com/FromDoppler/Doppler-UI-System/commit/4ed1095))
    * opacity disabled button, with no background color and no borders (d8ae371 (https://github.com/FromDoppler/Doppler-UI-System/commit/d8ae371))
    * Paragraphs were adjusted in their width and line-height to improve their readability (e1fa109 (https://github.com/FromDoppler/Doppler-UI-System/commit/e1fa109))
    * position background image (189ca57 (https://github.com/FromDoppler/Doppler-UI-System/commit/189ca57))
    * preload improvement (eabb2b2 (https://github.com/FromDoppler/Doppler-UI-System/commit/eabb2b2))
    * preload improvement (2bada01 (https://github.com/FromDoppler/Doppler-UI-System/commit/2bada01))
    * signup: add styles for the signup (42459db (https://github.com/FromDoppler/Doppler-UI-System/commit/42459db))
    * remove class button--loading (f39fbe9 (https://github.com/FromDoppler/Doppler-UI-System/commit/f39fbe9))
    * remove line break (10100dc (https://github.com/FromDoppler/Doppler-UI-System/commit/10100dc))
    * signup:  modify lables in signup layout and styles to optimize html (7efe940 (https://github.com/FromDoppler/Doppler-UI-System/commit/7efe940))
    * support general form error messages (19a959d (https://github.com/FromDoppler/Doppler-UI-System/commit/19a959d))
    * signup: add signup to the header (c211904 (https://github.com/FromDoppler/Doppler-UI-System/commit/c211904))
    * style: have been modified the properties of the text so that they are uppercase (2fd9793 (https://github.com/FromDoppler/Doppler-UI-System/commit/2fd9793))
    * remove link footer (1a8c6de (https://github.com/FromDoppler/Doppler-UI-System/commit/1a8c6de))
    * remove margin (461671e (https://github.com/FromDoppler/Doppler-UI-System/commit/461671e))
    * remove uppercase from user menu (92abc73 (https://github.com/FromDoppler/Doppler-UI-System/commit/92abc73))
    * remplace class (6bd3996 (https://github.com/FromDoppler/Doppler-UI-System/commit/6bd3996))
    * rename class (54e478c (https://github.com/FromDoppler/Doppler-UI-System/commit/54e478c))
    * separation of buttons in modal form (29c1c72 (https://github.com/FromDoppler/Doppler-UI-System/commit/29c1c72))
    * show notification message (792a50d (https://github.com/FromDoppler/Doppler-UI-System/commit/792a50d))
    * Styles for forgot password (1b14bf7 (https://github.com/FromDoppler/Doppler-UI-System/commit/1b14bf7))
    * support including buttons with link style inside error messages (81240de (https://github.com/FromDoppler/Doppler-UI-System/commit/81240de))
    * style: add hover in link (a122a5a (https://github.com/FromDoppler/Doppler-UI-System/commit/a122a5a))
    * the class field-group and field-item was added. (840aa20 (https://github.com/FromDoppler/Doppler-UI-System/commit/840aa20))
    * the classes were added to show the errors and variable for margin normalization (662314b (https://github.com/FromDoppler/Doppler-UI-System/commit/662314b))
    * signup: delete class holder-policy that no use (6d6b9af (https://github.com/FromDoppler/Doppler-UI-System/commit/6d6b9af))
    * the font size of the menu was modified (d08a50d (https://github.com/FromDoppler/Doppler-UI-System/commit/d08a50d))
    * variable was added for font size (d18e3f8 (https://github.com/FromDoppler/Doppler-UI-System/commit/d18e3f8))
    * visual adjustment in sign up (07c5030 (https://github.com/FromDoppler/Doppler-UI-System/commit/07c5030))
    * style: modify font-size footer (b4816f1 (https://github.com/FromDoppler/Doppler-UI-System/commit/b4816f1))
    * visual error in upgrade submit (b457a6d (https://github.com/FromDoppler/Doppler-UI-System/commit/b457a6d))
    * styles: separate this component in form (91c0bbb (https://github.com/FromDoppler/Doppler-UI-System/commit/91c0bbb))
    * width adjustment (508dea4 (https://github.com/FromDoppler/Doppler-UI-System/commit/508dea4))
    * signup: add the html structure of the form (648b489 (https://github.com/FromDoppler/Doppler-UI-System/commit/648b489))
    * signup: location folder image for logo doppler (f8f9e6f (https://github.com/FromDoppler/Doppler-UI-System/commit/f8f9e6f))
    * signup: modifications in the structure and checkbox (4218188 (https://github.com/FromDoppler/Doppler-UI-System/commit/4218188))
    * signup: modify class and structure in feature layout (5d461f9 (https://github.com/FromDoppler/Doppler-UI-System/commit/5d461f9))
    * signup: modify class name for more semantic (5896b26 (https://github.com/FromDoppler/Doppler-UI-System/commit/5896b26))
    * signup: modify structure and class signup layout for optimize html (ad60f06 (https://github.com/FromDoppler/Doppler-UI-System/commit/ad60f06))
    * style: add new buttons component (48231b5 (https://github.com/FromDoppler/Doppler-UI-System/commit/48231b5))
    * style: remplace scroll property for auto that no force scrollbar (28c7764 (https://github.com/FromDoppler/Doppler-UI-System/commit/28c7764))
    * styles: add signup layout (cbdaadb (https://github.com/FromDoppler/Doppler-UI-System/commit/cbdaadb))
    * styles: separate this component in form (fb9af0b (https://github.com/FromDoppler/Doppler-UI-System/commit/fb9af0b))

### BREAKING CHANGES

    * show notification message
    * add "button" element instead "link" element in user-menu
```

### CODE REVIEW

**Comment by IgnacioRodrigues** [comment]
Excelent work @andresmoschini the information of each version is really important, I really liked this new improvement.

**Comment by andresmoschini** [comment]
I am merging, sadly I cannot test it without merge, so if something is wrong I will need to fix it directly on master.

File: build-n-release-w-docker.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    -e GH_TOKEN \
+    -e "NPM_TOKEN=00000000-0000-0000-0000-000000000000" \
+    -v `pwd`:/work \
+    -w /work \
+    node:10 \
+    /bin/sh -c "\
+      npm install -g gulp \
+      && npm install \
+      && gulp dist \
+      && ./node_modules/.bin/semantic-release \
```
Comment (cbernat) [inline build-n-release-w-docker.sh]: ❤️

**Pull request merged**